pkg/chunk/cache/memcached_test.go

```diff
@@ -151,3 +153,45 @@ func testMemcacheFailing(t *testing.T, memcache *cache.Memcached) {
 		}
 	}
 }
+
+// TestUpdateLoop simulates memcachedClient.updateLoop and verifies its completeness
```

Comment:
Hrm, this test simulates it but is not a good unit test. For example, if in the future, if we change the logic of the code in `updateLoop`, we will never know. I don't think having this test adds any value, and if testing the `updateLoop` itself is too much, lets just chuck the test.